sdk/keyvault/azure-security-keyvault-common/src/keyvault_exception.cpp

```diff
@@ -0,0 +1,67 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+#include "azure/keyvault/common/keyvault_exception.hpp"
+#include "azure/keyvault/common/keyvault_constants.hpp"
+
+#include <azure/core/http/policy.hpp>
+
+#include <nlohmann/json.hpp>
+#include <type_traits>
+
+using namespace Azure::Security::KeyVault::Common;
+
+namespace {
+
+inline std::string GetHeaderOrEmptyString(
```

Comment:
This could be helpful on the request object.  We should move to Request so that users can do
`request->GetHeader(headerName)`